controller/store/annotations.go

```diff
@@ -141,6 +141,7 @@ var defaultAnnotationValues = MapStringW{
 	"ssl-redirect-code":       &StringW{Value: "302"},
 	"ssl-passthrough":         &StringW{Value: "false"},
 	"server-ssl":              &StringW{Value: "false"},
+	"enable-grpc":             &StringW{Value: "false"},
```

Comment:
Do we really need default value if it's `false` @Mo3m3n?

I'd use the default annotations in case of non default values, as `true` for booleans or non empty string for `strings`.

Reply:
it can't have default value, if it does it will override what  'server-ssl' annotation is doing